eva-lib/src/main/java/uk/ac/ebi/eva/lib/filter/FilterBuilder.java

```diff
@@ -0,0 +1,84 @@
+package uk.ac.ebi.eva.lib.filter;
+
+import org.opencb.biodata.models.variant.Variant;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FilterBuilder {
+
+    private List<VariantEntityRepositoryFilter> filters = new ArrayList<>();
+
+    public List<VariantEntityRepositoryFilter> getEvaWsVariantEntityRepositoryFilters(String maf,
+                                                                                      String polyphenScore,
+                                                                                      String siftScore,
+                                                                                      List<String> studies,
+                                                                                      List<String> consequenceType) {
+        return this.withMaf(maf)
+                   .withPolyphenScore(polyphenScore)
+                   .withSiftScore(siftScore)
+                   .withStudies(studies)
+                   .withConsequenceType(consequenceType)
+                   .build();
+    }
+
+    public List<VariantEntityRepositoryFilter> build() {
+        return filters;
+    }
+
+    public FilterBuilder withMaf(String maf) {
+        if (maf != null){
```

Comment:
Please auto-format the code, there should be a whitespace after the parenthesis.